web/app/Http/Controllers/JobsController.php

```diff
@@ -17,9 +18,23 @@ class JobsController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
-      $jobs = Jobs::orderBy('created_at', 'desc')->get();
+      $jobs = [];
+      $type = $request->get('type');
+      if($type && $type === 'need_approval') {
```

Comment:
`$type && ` can be removed because if `type` is not set in the `$request`, it would default to null so it won't through error.